data_service/adapters/storage/local.py

```diff
@@ -14,13 +14,13 @@ def __init__(self, settings: config.LocalFileSettings = Depends(get_settings)):
         self.log = logging.getLogger(__name__ + '.LocalFileAdapter')
         self.settings = settings
 
-    def get_file(self, dataset_name: str, version: str) -> str:
-        dataset_version = version.replace('.', '_')[:3]
-        return self.__create_download_path(dataset_name, dataset_version)
+    def get_file_path(self, path: str) -> str:
+        return f"{self.settings.DATASTORE_DIR}/data/{path}"
 
-    def __create_download_path(self, dataset_name: str, version: str) -> str:
-        path = (
-            f"{self.settings.DATASTORE_DIR}/data/"
-            f"{dataset_name}/{dataset_name}__{version}"
+    def get_data_versions(self, version: str) -> str:
+        version_underscored = version.replace('.', '_')[:3]
```

Comment:
We have to remember to not bother making a new "data_versions"-file on bump if the bump is only a PATCH

Reply:
If last datastore version is i.e. 1.2.0 and you bump a PATCH then the new datastore version will be 1.2.1.
Won't we make a data_versions__1_2_1 then?

Reply:
Yes, I agree with @vaskalan. We need to create it for every version that the client can query.

Reply:
The naming standard should be `data_versions__1_0.json` if we decide not to crate a new data_versions feil for PATCH.

Reply:
Yes, most probably there will not be a case for PATCH with new/changed/revised data. With this change the bumping will need to check bump type and create these data-versions files only if type is ADD or CHANGE_DATA. There will be fewer `data_versions__x_x.json` files though.

Reply:
or bump type is MAJOR or MINOR

Reply:
Yes, I agree its much cleaner to generate one for each datafile. I initially thought we were not going for this based on the data files themselves having just __MAJOR_MINOR in their file name, and the line of code this thread is connected to.
Generating one data_version__M_m_p.json for each bump regardless of bump-type seems like the cleanest/simplest solution though, so I absolutely agree.